common/array/index.js

```diff
@@ -165,7 +166,7 @@ const ListArray = ({
 
   return (
     <div
-      className={classNames('lf-control-common-array', align ? `lf-${align}` : undefined)}
+      className={classNames('lf-control-common-array', className, align ? `lf-${align}` : undefined)}
```

Comment:
Thanks for the PR. I don't think we should put the className here, it's already set in the per-framework container of the common array, for example here

https://github.com/guidone/lets-form/blob/main/react-antd/array/index.js

Reply:
yes you're right, sorry for the delay, was knocked out by a flu.

Merging, thanks for the feedback 🖖